api/server/router/container/container_routes.go

```diff
@@ -372,6 +372,14 @@ func (s *containerRouter) postContainersCreate(ctx context.Context, w http.Respo
 	version := httputils.VersionFromContext(ctx)
 	adjustCPUShares := version.LessThan("1.19")
 
+	// set container policies associated by authz plugin(s)
+	policies, ok := ctx.Value("policies").(map[container.PolicyType]string)
+	if !ok {
+		logrus.Warnf("incorrect type for policy map %T, expected %T", ctx.Value("policies"), config.Policies)
```

Comment:
this is just a log message for type assertion failure. 

Yes, it is ok for no policies to be associated. If policies are not associated then we just set the map as empty on the next line so that it is not nil.